### ISSUE TITLE
Support for non-base types?

### ISSUE THREAD

**Issue by ses4j** [yukinarit/pyserde#4]
I use `decimal.Decimal` frequently, and I want to serialize to yaml.  Is there an extensibility feature that will enable this to serialize?  Currently I get messages like:

```
yaml.representer.RepresenterError: ('cannot represent an object', Decimal('75000'))
```

**Comment by yukinarit** [comment]
Hi @ses4j 
Sorry, I was completely missing out this issue. I am going to push the support for non-base types in the next few days. Thanks!

**Comment by yukinarit** [comment]
@ses4j 
I believe decimal support has been released as [0.0.9](https://pypi.org/project/pyserde/), and Here is the [test case](https://github.com/yukinarit/pyserde/blob/master/tests/test_more_types.py#L10-L23). Please Let me know if this doesn't work with your code. Thank you.
